pkg/process/manager.go

```diff
@@ -191,6 +203,57 @@ type Opt struct {
 	Database         *store.Database
 	DaemonMode       string
 	CacheDir         string
+	RecoverPolicy    string
+}
+
+func (m *Manager) handleDaemonDeathEvent() {
+	for event := range m.LivenessNotifier {
```

Comment:
It's a time-consuming operation when handling each event, do we need to put it in gorountine? otherwise it will cause many events to be blocked once a daemon hangs.

Reply:
Good idea 👍

Reply:
fixed